chapters/en/slides/module3/module3_01.Rmd

```diff
@@ -0,0 +1,274 @@
+rather than looking at individual points.
+
+This type of plot with vertical tick marks along one axis
+is so common that it is sometimes referred to by a special name: rugplot.
+
+The visualization grammar of Altair let's us think about it more intuitively
+by composing it with a mark and a single encoding,
+instead of remembering a specific name.
+
+---
+
+## Thinner marks also saturate for large data sets
+
+```{python}
+alt.Chart(movies).mark_tick().encode(
+    x='runtime')
+```
+
+Notes:
+However,
+when we increase the number of observations even further (to about 500)
+it becomes difficult to tell exactly how the data distribution looks,
+even with these thin marks.
+
+---
+
+## Using histograms to visualize distributions avoids saturation issues
+
+```{python}
+alt.Chart(movies).mark_bar().encode(
+    alt.X('runtime', bin=alt.Bin(maxbins=30)),
+    y='count()')
+```
+
+Notes:
+Instead of visualizing each individual observation,
+we can use the histograms we learnt about in last module,
+where we divide one axis into groups (or "bins")
+and represent the count of observations in each group with bar.
+
+In this plot,
+we can directly see that most movies are around 100 min in length.
+But from our experience,
+we know that factors such as genre and production year,
+can influence movie length.
+
+How can we separate the data based on other columns in the dataset
+and compare multiple distributions?
+
+---
+
+## Comparing multiple distributions via faceting
+
+```{python}
+(alt.Chart(movies).mark_bar().encode(
+    alt.X('runtime', bin=alt.Bin(maxbins=30)),
+    y='count()')
+ .facet('country'))
+```
+
+Notes:
+Let's start by comparing the movie length broken down per country.
+Here we use faceting which we learnt about in the last module,
+to create one histogram per country.
+
+You can see that this dataset only contains movies made in the UK or the US.
+It does seem like the distribution of movies is pretty equal between the two countries,
+but it is a bit hard to see because there are so many more movies made in the US.
+
+---
+
+## Comparing multiple distributions via faceting with independent y-axes
+
+```{python}
+(alt.Chart(movies).mark_bar().encode(
+    alt.X('runtime', bin=alt.Bin(maxbins=30)),
+    y='count()')
+ .facet('country')
+ .resolve_scale(y='independent'))
+```
+
+Notes:
+By default when faceting,
+the y-axis is the same for all the subplots.
+This is often the most suitable choice,
+because it gives us an idea of both the shape of the distribution
+and the total number of observations.
+
+If we want to drill down and compare the distribution shape more closely
+regardless of the number of observations
+we could define the y-axes to be independent for each subplot,
+instead of linked to the same axis range.
+
+Now we can see that the distributions might be a bit different,
+it looks like the UK one is slightly bimodal (it has two peaks),
+whereas the US one is unimodal (it has one peak).
```

Comment:
These distributions do not look meaningfully different to me. The bimodalness you are commenting on is an artifact of the binning and low number of observations in the UK data. My conclusion would be that they are fairly similiar, maybe I would play with the binning on another slide to drive this point home to the learners (say bins of 15 or 10).

Reply:
Ok will look into this. Would be a good idea to play with binning either way